### ISSUE TITLE
Stuck on loading dependency for Framer

### ISSUE THREAD

**Issue by ray1claw** [jchavarri/framer-boilerplate#1]
During the initial `npm install`, the installer gets 
stuck at loading the Framer dependency from git. Attaching a screenshot below.

I tried `npm install framer` separately and it goes into `./npm_modules/framer` instead of `./npm_modules/framerjs` plus there are no build targets inside to initiate make. Is there anything I'm missing here?

![screenshot from 2015-11-26 13 18 00](https://cloud.githubusercontent.com/assets/149414/11417833/28df46aa-9440-11e5-8269-eb6281f48ae7.png)

**Comment by jchavarri** [comment]
Hi @ray1claw

Same thing happened to me. I don't know the exact reason, but it takes a really long time for npm to download the Framer package from Github. I thought it was stuck, but after some minutes (around 5, maybe) it finally gets downloaded and installed.

Could you check if it works properly when you leave the installer running a bit longer?

**Comment by ray1claw** [comment]
@jchavarri I waited around for a lot longer, more than a couple hours and it still was stuck there (but the speeds do suck here too).

Alternatively, I did the following:

```
git clone https://github.com/koenbok/Framer.git node_modules/framerjs
cd node_modules/framerjs
make
```

which resulted in errors about some deprecated dependencies and also some optional ones being unavailable. I had to close it then but trying it again now. Will report if anything comes up.

**Comment by jchavarri** [comment]
Thanks for the feedback. I just tried myself and it takes around 5 min but it finally downloads everything. Just so you get an idea, the whole `node_modules` folder weights around 230MB.

One question: which version of node / npm are you using? I have node v5.0.0 and npm v3.3.6. Apparently [there were some problems related with performance](https://github.com/npm/npm/releases/tag/v3.3.6) in the first v3 releases of npm.

**Comment by jchavarri** [comment]
@ray1claw Did you manage to solve this? I recently updated npm to the latest version and now downloading repos from a specific Github commit takes very few seconds.

**Comment by ray1claw** [comment]
@jchavarri yes! I tried it the other day with a faster Internet and it did successfully proceed further. 

There were a couple warnings about deprecated dependencies though during the compile (one was for latest version of phantomjs not supported, can't remember the other one). I'm using node v4.2.2 and npm v3.4.1.

The compile did successfully generate framer.js and I'm able to use the scaffold without any issues! I'm closing this issue as I wouldn't consider the warnings I encountered of any consequence.

Again, thanks for the help!
